### PR TITLE
Playground

### PR DESCRIPTION
**Pull request by CaroFG**

### CODE REVIEW

File: lib/meilisearch/pagination/kaminari.rb [hunk 1]
```diff
@@ -24,7 +24,7 @@ def offset(num)
 
       class << self
         def create(results, total_hits, options = {})
-          offset = ((options[:page] - 1) * options[:per_page])
+          offset = ((options[:page].to_i - 1) * options[:per_page])
```
Comment (curquiza) [inline lib/meilisearch/pagination/kaminari.rb:27]: Why does this file change in this PR?

**Review by curquiza — CHANGES_REQUESTED** [review]
Is it possible to display a browse of documents (placeholder search) on the search page? If I don't go to the seed, I don't know what kind of result I can search for.
Also, can we add a sentence to say we need to press "enter" to activate the search? 🙂 We can easily think this is broken because we are not get used to use the search bar this way now (internally).

File: lib/meilisearch/pagination/kaminari.rb (hunk 1 above)
Comment (CaroFG) [reply lib/meilisearch/pagination/kaminari.rb:27]: My bad

**Review by curquiza — CHANGES_REQUESTED** [review]
> Is it possible to display a browse of documents (placeholder search) on the search page? If I don't go to the seed, I don't know what kind of result I can search for.

<s>I still don't see this point 😢 </s> My bad, wrong manipulation, it's ok!

File: lib/meilisearch/pagination/kaminari.rb [hunk 2]
```diff
@@ -25,12 +25,12 @@ def offset(num)
       class << self
         def create(results, total_hits, options = {})
           offset = ((options[:page] - 1) * options[:per_page])
-          array = new results, :offset => offset, :limit => options[:per_page], :total_count => total_hits
+          array = new results, :limit => options[:per_page], :offset => offset
```
Comment (curquiza) [inline lib/meilisearch/pagination/kaminari.rb:28]: Why do we change this file in this PR?

Comment (CaroFG) [reply lib/meilisearch/pagination/kaminari.rb:28]: Pagination with Kaminari wasn't working, I didn't understand why, so I went through [their doc](https://www.rubydoc.info/gems/kaminari/0.12.3/Kaminari/PaginatableArray) and realized the last parameter wasn't necessary. When I removed it, pagination started working.

Comment (curquiza) [reply lib/meilisearch/pagination/kaminari.rb:28]: I've just added `total_count` at the end of the line and it works:
```
array = new results, :limit => options[:per_page], :offset => offset, :total_count => total_hits
```

File: playground/.gitignore [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+# Ignore master key for decrypting credentials and more.
+/config/master.key
+
+/public/packs
+/public/packs-test
+/node_modules
+/yarn-error.log
+yarn-debug.log*
+.yarn-integrity
```
Comment (curquiza) [inline playground/.gitignore:40]:
```suggestion
.yarn-integrity

# MeiliSearch
data.ms
```

Comment (curquiza) [reply playground/.gitignore:40]: To avoid pushing the data.ms 😇

**Review by curquiza — APPROVED** [review]
perfecto

merge with bors when you wnat :)

**Comment by CaroFG** [comment]
bors try

**Comment by CaroFG** [comment]
bors merge

**Pull request merged**